### PR TITLE
VEL3D Corrections

### PR DESCRIPTION
**Pull request by cwingard**
Updates the VEL3D tables for Endurance based on local testing; removing test variables that are not part of the data sets and correcting the climatology table so the values for CE07SHSM get tested.

### CODE REVIEW

File: qartod/vel3d/vel3d_qartod_gross_range_test_values.csv [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
 CE09OSPM,WFP01,01-VEL3DK000,vel3d_k_wfp_instrument,{'inp': 'vel3d_k_upward_velocity'},"{'qartod': {'gross_range_test': {'suspect_span': [-0.29644, 0.28489], 'fail_span': [-1.0, 1.0]}}}",User Gross Range based on data collected from 2014-04-19 through to 2023-12-31.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
-CE09OSPM,WFP01,01-VEL3DK000,vel3d_k_wfp_stc_instrument,{'inp': 'int_ctd_pressure'},"{'qartod': {'gross_range_test': {'suspect_span': [30.00, 515.00], 'fail_span': [0.0, 1000.0]}}}",Derived from mooring design and limits on MMP travel,Mooring design includes physical stops attached to the riser cable that limit how far up and down the MMP can travel.
```
Comment (cooleyky) [inline qartod/vel3d/vel3d_qartod_gross_range_test_values.csv:11]: The revised csv looks alright. While the test is not executed, this parameter is still in downloaded datasets. I think this needs a closer look

**Review by cooleyky — COMMENTED** [review]
Everything looks to be formatted correctly. I've asked for follow up on the deleted tests for `int_ctd_pressure` parameters

**Review by mvardaro — APPROVED** [review]
LGTM. int_ctd_pressure is a separate discussion.

File: qartod/vel3d/vel3d_qartod_climatology_test_values.csv [hunk 2]
```diff
@@ -21,10 +21,10 @@ CE07SHSM,MFD35,01-VEL3DD000,vel3d_cd_dcl_system_data,"{'inp': 'temperature_centi
...
+CE07SHSM,MFD35,01-VEL3DD000,vel3d_cd_dcl_velocity_data_recovered,"{'inp': 'vel3d_c_eastward_turbulent_velocity', 'tinp': 'time', 'zinp': 'None'}",climatology_tables/CE07SHSM-MFD35-01-VEL3DD000-velocity_east_corrected.csv,Climatology based on data collected from 2016-05-06 through to 2023-12-31.,The variance explained by the climatological model is 33.1%.
+CE07SHSM,MFD35,01-VEL3DD000,vel3d_cd_dcl_velocity_data_recovered,"{'inp': 'vel3d_c_northward_turbulent_velocity', 'tinp': 'time', 'zinp': 'None'}",climatology_tables/CE07SHSM-MFD35-01-VEL3DD000-velocity_north_corrected.csv,Climatology based on data collected from 2016-05-06 through to 2023-12-31.,The variance explained by the climatological model is 22.7%.
+CE07SHSM,MFD35,01-VEL3DD000,vel3d_cd_dcl_velocity_data_recovered,"{'inp': 'vel3d_c_upward_turbulent_velocity', 'tinp': 'time', 'zinp': 'None'}",climatology_tables/CE07SHSM-MFD35-01-VEL3DD000-velocity_vertical.csv,Climatology based on data collected from 2016-05-06 through to 2023-12-31.,The variance explained by the climatological model is 12.5%.
```
[the remark this answers is not in the document]
Comment (cwingard) [reply qartod/vel3d/vel3d_qartod_climatology_test_values.csv:27]: Removed the changes to int_ctd_pressure. Still need these changes approved and merged.

**Review by cooleyky — APPROVED** [review]
looks good

**Pull request merged**